README.md

```diff
@@ -21,7 +21,199 @@ ember install ember-frost-bunsen
 Coming soon…
 
 ## Examples
-Coming soon
+
+### Form View
+
+```handlebars
+{{
+  frost-bunsen-form
+  model=model
+  view=view
+}}
+```
+
+### Detail View
+
+```handlebars
+{{
+  frost-bunsen-detail
+  model=model
+  value=value
+  view=view
+}}
+```
+
+> Note: ALL values, models, and views MUST be valid [JSON](http://www.json.org/). Values are simply the data being represented in the UI which usually come directly from an API response. Models must be valid [JSON Schema](http://json-schema.org/) and views must be valid [view schema](https://github.com/ciena-frost/ember-frost-bunsen/blob/master/addon/components/validator/view-schema.js). Below we will provide examples of values, models, and views to give you a better idea of how this stuff works.
+
+### Minimal Example
+
+**Value**
```

Comment:
Value is output, correct?  I'd suggest indicating this as a result since it looks like it's part of the interface config currently